src/application/app.js

```diff
@@ -39,7 +39,7 @@ function checkInputAndConfig(config, fromDate, toDate) {
 }
 
 async function mcodeApp(Client, fromDate, toDate, pathToConfig, pathToRunLogs, debug, allEntries) {
-  if (debug) logger.level = 'debug';
+  logger.level = (debug ? 'debug' : 'info');
```

Comment:
I don't think this makes a difference functionally and it doesn't seem to be a lint rule, but I think we tend to not wrap ternary operators in `()` unless it's necessary. I don't think we need them here, so the suggestion below should be sufficient!

```suggestion
  logger.level = debug ? 'debug' : 'info';
```